### PR TITLE
Added few images to build

### PR DESCRIPTION
**Pull request by aleksuss**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -43,12 +46,12 @@ environment:
...
-  # Beta 64-bit MSVC
-    - channel: beta
-      target: x86_64-pc-windows-msvc
-  # Beta 32-bit MSVC
-    - channel: beta
-      target: i686-pc-windows-msvc
+#  # Beta 64-bit MSVC
+#    - channel: beta
+#      target: x86_64-pc-windows-msvc
+#  # Beta 32-bit MSVC
```
Comment (stanislav-tkach) [inline appveyor.yml:52]: Should this comments be removed?

File: librocksdb-sys/build.rs [hunk 2]
```diff
@@ -2,9 +2,9 @@ extern crate cc;
 extern crate pkg_config;
 
 use pkg_config::probe_library;
-use std::process::Command;
-use std::fs::read_dir;
 use std::env::var;
+use std::fs::read_dir;
+use std::process::Command;
```
Comment (stanislav-tkach) [inline librocksdb-sys/build.rs:7]: How about using import groups? :upside_down_face:

**Review by stanislav-tkach — APPROVED** [review]

File: appveyor.yml (hunk 1 above)
Comment (aleksuss) [reply appveyor.yml:52]: Actually I plan to use this properties in nearest future. So let it be 👌 ?

File: librocksdb-sys/build.rs (hunk 2 above)
Comment (aleksuss) [reply librocksdb-sys/build.rs:7]: Maybe in another dedicated PR 😃 Or rustfmt will do this in the future for us. I really hope for it 😸

**Pull request merged**